### PR TITLE
Enable Zuul

### PR DESCRIPTION
**Pull request by gundalow**
Now we are happy with how network-engine-zuul has been performing, we
can make Zuul live on the main development repo

### CODE REVIEW

**Comment by gundalow** [comment]
For once https://softwarefactory-project.io/r/#/c/12279/ is merged

**Comment by gundalow** [comment]
recheck

File: .zuul.yaml [hunk 1]
```diff
@@ -1,4 +1,13 @@
 - project:
     check:
       jobs:
-        - noop
+        - ansible-test-sanity
+        - ansible-role-tests-stable-py2
+        - ansible-role-tests-stable-py3
+        - ansible-role-tests-devel-py2
+    gate:
```
Comment (trishnaguha) [inline .zuul.yaml:8]: We would not like to enable `gate` for the repo.

**Review by trishnaguha — CHANGES_REQUESTED** [review]

File: .zuul.yaml (hunk 1 above)
Comment (gundalow) [reply .zuul.yaml:8]: For a PR to get into the `gate` pipeline it needs to:
* pass CI
* have at least one +1
* have no -1
* be given a `mergeit` GitHub label

https://github.com/ansible-network/zuul-config/blob/master/zuul.d/pipelines.yaml#L50

So nothing should be ending up in that pipeline

Comment (trishnaguha) [reply .zuul.yaml:8]: That sounds great. thanks for the explanation!

**Review by trishnaguha — APPROVED** [review]

**Pull request merged**